
Fix unresolved external "fiprintf".
When newlib is configured with:

  --enable-newlib-nano-formatted-io

"fiprintf" should be an alias of "fprintf". However the alias does 
not seem to work properly, as we encounter unresolved external
error if we uses "assert" in code. So we patch the code directly.

Signed-off-by: Juro Bystricky <juro.bystricky@intel.com>

--- a/newlib/libc/stdlib/assert.c	2016-11-08 08:04:45.669248139 -0800
+++ b/newlib/libc/stdlib/assert.c	2016-11-08 08:05:47.357554034 -0800
@@ -56,7 +56,7 @@
 	const char *func _AND
 	const char *failedexpr)
 {
-  fiprintf(stderr,
+  fprintf(stderr,
 	   "assertion \"%s\" failed: file \"%s\", line %d%s%s\n",
 	   failedexpr, file, line,
 	   func ? ", function: " : "", func ? func : "");
